src/electron/main.ts

```diff
@@ -37,6 +39,12 @@ function createWindow() {
         mainWindow = null;
     });
 
+    // Provides a more graceful experience and eliminates the white screen on load
```

Comment:
Also helps to reduce white screen on app launch - Only show the main window after the first app render